Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -9,19 +9,35 @@
 import UIKit
 
 class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelegate {
-    @IBOutlet weak var headerText: UILabel!
-    @IBOutlet weak var tfPassword: UITextField!
-    @IBOutlet weak var tfConfirmPassword: UITextField!
-    @IBOutlet weak var lbMatchIndicator: UILabel!
+    // Input fields
```

Comment:
The variables can be renamed instead to more clearly indicate that these are input fields rather than adding a comment. Some codes can be/should be self-documenting.

Reply:
resolved in ab83980